src/components/common/mes-back-button/mes-back-button.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, Input } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'mes-back-button',
+  templateUrl: 'mes-back-button.html',
+  styleUrls: ['./mes-back-button.scss'],
+})
+export class MesBackButtonComponent {
+
+  @Input()
+  public routerLink: string;
+  public buttonDisabled = false;
+
+  constructor(
+    private router: Router,
+  ) { }
+
+  public async onClick(event: Event) {
```

Comment:
:promise<void> as return value?

Reply:
hmm, apparently i cant write promise < void >

Reply:
not sure what you mean sorry